core/SConscript.firmware

```diff
@@ -428,6 +441,7 @@ env.Replace(
         'FIRMWARE',
         'TREZOR_MODEL_'+TREZOR_MODEL,
         'USE_HAL_DRIVER',
+        LAYOUT,
```

Comment:
Nit: not sure layout is the right word here. What do you think about using `UI` instead?

Reply:
i can see the problem with `LAYOUT`, but `UI` seems a bit too general. how about `UI_LAYOUTS`, `UI_TYPE` or something like that?

Reply:
Both sound fine. Either `UI_LAYOUTS` or `UI_LAYOUT`?

Reply:
`UI_LAYOUT` it is